contracts/src/v0.8/functions/dev/v1_X/accessControl/interfaces/ITermsOfServiceAllowList.sol

```diff
@@ -69,6 +69,11 @@ interface ITermsOfServiceAllowList {
     uint64 blockedSenderIdxStart,
     uint64 blockedSenderIdxEnd
   ) external view returns (address[] memory blockedSenders);
+
+  /// @notice Syncs the entire allowlist from the previous v1.0.0 ToS contract to the new v1.1.1 contract
```

Comment:
Do we need to mention version numbers here?

Reply:
no. Will remove.